packages/dredd/LICENSE

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Does this mean there is a license in the root of the repository and Lerna distributes it with each package?

Reply:
Looks like you've implemented this in Lerna: https://github.com/lerna/lerna/issues/1213#issuecomment-392548883

I'd need to verify how the package is distributed.

Reply:
Yeah :D the distributing. But I didn't even run Lerna for real, I just wrote tests and sent a PR :D